### ISSUE TITLE
QA Ph3_Accessibility: Schedule - Issues navigating pages with keyboard only 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#625]
There are a number of issues when navigating the schedule pages with the keyboard only.  These issues are reproducible on each of the subnav options (Agenda, May 18, May 19, May 20, My Schedule). 
-All sub nav options are skipped with the exception of that which is currently landed on.  For example, if user is landed on the May 18 tab (which is the initial landing location for the Schedule page), they will be unable to access Agenda, May 19, May 20, or My Schedule.
-There is lag time after the user has tabbed to Filter and before they land on the first listed event.  

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Select Schedule 
3. Use the keyboard only to navigate the page 
4. Continue tapping the tab key to observe the skipped links and lag time after Filter 

Browser: 
Chrome 49 Mac OS X  10.11 

![gio_625](https://cloud.githubusercontent.com/assets/15128124/14442163/1ebfbd8a-ffed-11e5-8fdb-bb3d06a6acf1.png)
